src/nupic/algorithms/SDRClassifier.hpp

```diff
@@ -33,20 +34,30 @@
 #include <string>
 #include <vector>
 
-#include <nupic/types/ClassifierResult.hpp>
 #include <nupic/types/Types.hpp>
 #include <nupic/types/Serializable.hpp>
 
 namespace nupic {
 namespace algorithms {
 namespace sdr_classifier {
-using namespace std;
-using nupic::types::ClassifierResult;
+
 
 const UInt sdrClassifierVersion = 2;
```

Comment:
probably should bump classifier version, not that we use it anyhow

Reply:
This is used for serialization and the save/load format does not change in this PR.